src/main/java/io/stargate/sgv2/jsonapi/api/model/command/builders/TableSortClauseBuilder.java

```diff
@@ -14,4 +21,50 @@ public TableSortClauseBuilder(TableSchemaObject table) {
   public SortClause buildAndValidate(ObjectNode sortNode) {
     return defaultBuildAndValidate(sortNode);
   }
+
+  @Override
+  protected String validateSortClausePath(String path) {
+    // Tables have few rules: but cannot be empty
+    if (path.isEmpty()) {
+      throw ErrorCodeV1.INVALID_SORT_CLAUSE_PATH.toApiException(
+          "path must be represented as a non-empty string");
+    }
+    // Do we use escaping for Tables? For sake of consistency I think so
+    try {
+      path = DocumentPath.verifyEncodedPath(path);
+    } catch (IllegalArgumentException e) {
+      throw ErrorCodeV1.INVALID_SORT_CLAUSE_PATH.toApiException(
+          "sort clause path ('%s') is not a valid path. " + e.getMessage(), path);
+    }
+
+    return path;
+  }
+
+  @Override
+  protected SortExpression buildAndValidateExpression(
+      String validatedPath, JsonNode innerValue, int totalFields) {
+    float[] vectorFloats = tryDecodeBinaryVector(validatedPath, innerValue);
```

Comment:
a bit unclear here, comments would help.Are we testing for a vector sort first because it is easier to test for ?